CONTRIBUTING.md

```diff
@@ -43,7 +45,11 @@
 ├── package.json
 ```
 
-# How can you make your first **Pull Request**
+# How can you make a **Pull Request** (PR)
+
+If you get an issue assigned to you, consider following steps mentioned below.
+
+**Note**: Steps 1, 2 and 3 are **one-time** steps required for setup. If you have already cloned the repo and added upstream, consider following this documentation from step 4.
 
 1. **Forking repository**
```

Comment:
Can you add `description` for `Forking repository` for this website from the reference given along with the image ?
Like this but for `crypto-site`:

![image](https://user-images.githubusercontent.com/35677839/152313383-bf01f8c6-b584-4604-984d-928fb357d8df.png)